### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by FledgeXu**
Related to #279 
This PR may need some tests before it's merged.

### CODE REVIEW

**Comment by FledgeXu** [comment]
I tested the `--redis-url` and I didn't find any bugs.

**Review by rgaudin — COMMENTED** [review]
Thank you ; wouldn't close #279 though but the rest will be handled through the org-wide task first.
Please change the mistune version.

File: requirements.txt [hunk 1]
```diff
@@ -1,19 +1,19 @@
...
+python-slugify>=8.0.1,<9.0.0
+jinja2>=3.1.0,<3.2
+redis>=4.5.1,<5.0
 beautifulsoup4>=4.9.3,<5.0
 lxml>=4.9.1,<4.10
 jinja2-pluralize>=0.3.0,<0.4
-tld>=0.12.6,<0.13
-mistune>=2.0.0rc1,<2.1
+tld>=0.13,<0.14
+mistune>=3.0.0rc4,<3.1
```
[the remark this answers is not in the document]
Comment (FledgeXu) [reply requirements.txt:14]: You mean we don’t need to update `tld` and `mistune`, those two packages?

[the remark this answers is not in the document]
Comment (rgaudin) [reply requirements.txt:14]: No tld is fine but mistune's new version is to the next major version while it is somewhat critical. Given there's no changelog to look at differences, let's keep it in 2.x

File: requirements.txt [hunk 2]
```diff
@@ -1,19 +1,19 @@
 kiwixstorage>=0.8.1,<1.0
 pif>=0.8.2,<0.9
-zimscraperlib>=1.8.0,<1.9
+zimscraperlib>=2.0,<2.9
```
[the remark this answers is not in the document]
Comment (FledgeXu) [reply requirements.txt:3]: I will fix it than.

File: requirements.txt (hunk 1 above)
[the remark this answers is not in the document]
Comment (FledgeXu) [reply requirements.txt:14]: Ok, i will fix it.

**Comment by FledgeXu** [comment]
I update the `requirements.txt`, based on the previous discussion.

**Review by rgaudin — APPROVED** [review]
👍

**Pull request merged**